controllers/usersController.js

```diff
@@ -1,3 +1,38 @@
+const { User } = require('../services/authService');
+const { oneTimePass } = require('../services/registerService');
+
 const getUsers = async (req, res) => res.sendStatus(200);
 
-module.exports = { getUsers };
+async function handleGetUsers(req, res) {
+  const showAllUser = await User.find({});
+  return res.send(JSON.stringify(showAllUser));
+}
+
+async function handleGetUser(req, res) {
+  const user = await User.retrieve({ email: req.params.email });
+  return res.send(JSON.stringify(user));
+}
+
+async function handleUpdateUser(req, res) {
+  try {
+    console.log(req.body);
```

Comment:
do not forget to remove the redundant `console.log` before the production release.